lab_solutions/lab02_14_l2_disctance.py

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2020 Attila Nagy <attila.nagy234@gmail.com>
+#
+# Distributed under terms of the MIT license.
+
+
+def l2_distance(vector1, vector2):
+    dim1 = len(vector1)
+    dim2 = len(vector2)
+    dim_diff = abs(dim1 - dim2)
+    if dim_diff != 0:
+        # Pad with zeros if dim is not the same
+        if dim1 > dim2:
+            vector2.extend([0 for x in range(dim_diff)])
+        else:
+            vector1.extend([0 for x in range(dim_diff)])
```

Comment:
Don't do this. L2 is defined for vectors of the same dimension. You can just raise a ValueError instead.